src/components/AppToolbar.tsx

```diff
@@ -95,7 +97,7 @@ type AppToolbarProps = {
   renderer?: string
 };
 
-export default class AppToolbar extends React.Component<AppToolbarProps> {
+class IAppToolbar extends React.Component<AppToolbarProps & WithTranslation> {
```

Comment:
Can `& WithTranslation` be added to `AppToolbarProps` instead?

Reply:
If this is internal as you described in the comment, I would call it `AppToolbarInternal`, the prefix for `I` is usually used for interface and it causes an unneeded confusion.

Reply:
Got it. I'll rename everything to `~Internal` for the internal components